public/test.html

```diff
@@ -101,8 +101,7 @@
       </footer>
 
     </div>
-    <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/parse/dist/parse.min.js"></script>
```

Comment:
I think we should specify a version here?
```suggestion
    <script src="https://cdn.jsdelivr.net/npm/parse@5.3.0/dist/parse.min.js"></script>
```